.circleci/config.yml

```diff
@@ -226,6 +233,54 @@ jobs:
       - store_artifacts:
           path: TwilioPasskeys.aar
 
+  publish-kotlin-multiplatform-sdk-to-nexus-internal-repository:
+    executor: android-executor
+
+    steps:
+      - checkout
+      - android/restore-build-cache
+      - android/restore-gradle-cache
+      - run:
+          name: Generate signing key ring file
+          command: |
+            echo $SIGNING_KEY | base64 -d >> $SIGNING_SECRET_KEY_RING_FILE
+      - run:
+          name: Publish SDK to internal repository
+          command: ./gradlew sonatypeTwilioPasskeysStagingRepositoryUpload | tee gradle-task-output.log
+      - run:
+          name: Save internal repository link
+          command: ./scripts/save_sonatype_repository_internal_release_url.sh gradle-task-output.log
+      - persist_to_workspace:
+          root: *workspace
+          paths:
+            - tmp/workspace/env_vars
+
+  generate-android-sample-app-using-internal-sdk:
+    executor: android-node-executor
+
+    steps:
+      - checkout
+      - android/restore-build-cache
+      - android/restore-gradle-cache
+      - attach_workspace:
+          at: *workspace
+      - run:
+          name: Set sample backend URL
+          command: ./scripts/update_gradle_properties.sh sampleBackendUrl \"${SAMPLE_BACKEND_URL}\" androidApp/gradle.properties
+      - run:
+          name: Generate APK with maven repo url
+          command: |
+            source tmp/workspace/env_vars
+            echo $REPO_NAME
+            echo $REPO_URL
+            ./gradlew :androidApp:assembleRelease -PmavenUsername=$OSSRH_USERNAME -PmavenPassword=$OSSRH_PASSWORD -PmavenRepoUrl=$REPO_URL
+      - run:
+          name: Install Firebase CLI
+          command: npm install --save-dev firebase-tools
+      - run:
+          name: Deploy to Firebase App Distribution
+          command: firebase appdistribution:distribute androidApp/build/outputs/apk/release/androidApp-release.apk --app 1:702410982038:android:e26dad2baf4fe5b09278b7 --token $FIREBASE_TOKEN --groups qa --debug
```

Comment:
Should the value `1:702410982038:android:e26dad2baf4fe5b09278b7` be a variable in CircleCI?

Reply:
Done, moved to env variable